### ISSUE TITLE
importing a partitioned table generates an ERROR

### ISSUE THREAD

**Issue by Hanane-Amghar** [darold/ora2pg#1295]
Hello, 
I'm new to ora2pg, I'm using postgres v11.7 and ora2pg v19.1.
While trying to import tables from my Oracle database , I had this ERROR: 
$psql -d prod -U postgres -f /tmp/data_create_RRR.sql
SET
psql:/tmp/data_create_RRR.sql:15: ERROR:  syntax error at or near "("
LINE 6: ) PARTITION BY  () ;
                        ^
--> The Table in question is this : 
CREATE TABLE ref (
        id bigint,
        name varchar(200),
        type varchar(2),
        date timestamp
) PARTITION BY  () ;

--> Also , these directives are enabled in my ora2pg.conf : 
COMPILE_SCHEMA  1
PREFIX_PARTITION  1
DISABLE_PARTITION 1
PG_SUPPORTS_PARTITION 1

*** Could you please help me with this? 
Is there a way to avoid having a partitioned table imported to my postgres database without having to delete the "Partition BY" statement in my oracle database ?

**Comment by darold** [comment]
Please use latest Ora2Pg v23.0 this have been fixed in one of the previous release.

**Comment by Hanane-Amghar** [comment]
Thank you for your reply.
In ora2pg v23.0 will I need to enable/disable any directives including the ones stated above : ) ?

**Comment by darold** [comment]
Actually with DISABLE_PARTITION enabled you should not have the partition definition. If this is not the case there is a bug.

The PG_SUPPORTS_* directive are not more supported, just set PG_VERSION to 11 in your ora2pg.conf file instead, Ora2Pg will know what is supported or not in your PG destination database.

**Comment by Hanane-Amghar** [comment]
Yes in ora2pd v19.1 I enabled DISABLE_PARTITION  to 1 and it didn't delelte the partition definition . 
Does the directive PG_VERSION  only exists in ora2pg v23.0 ?

**Comment by Hanane-Amghar** [comment]
Yes in ora2pd v19.1 I enabled DISABLE_PARTITION to 1 and it didn't delete the partition definition .
Does the directive PG_VERSION only exist in ora2pg v23.0 ?

**Comment by darold** [comment]
No, it exists since version 20.0

**Comment by Hanane-Amghar** [comment]
Alright, Thank you

**Comment by Hanane-Amghar** [comment]
Does ora2pg v23.0 support postgres client v9.x  and DBD-Oracle-1.75_2 ?

**Comment by darold** [comment]
I would say try it, but yes they are fully supported.

**Comment by Hanane-Amghar** [comment]
Will do, thank you
